### PR TITLE
fix: Call for speaker enable field

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/5315
continue - https://github.com/fossasia/open-event-server/pull/7602

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Why do you close and reopen the PRs, you can always force push on the same branch

**Comment by iamareebjamal** [comment]
Migration needs to enable CFS for events which have s CFS entry

**Comment by maze-runnar** [comment]
> you can always force push on the same branch

I do that in frontend, but here I often have issues that I couldn't understand, one of them is revision identified by 'acef2123' is missing 🤦🏻‍♂️  . 

> Migration needs to enable CFS for events which have s CFS entry


```
UPDATE events
SET    is_cfs_enabled = True,
FROM   events
       JOIN speakers_calls
         ON events.id = speakers_calls.event_id
WHERE
       speakers_calls.announcement IS NOT NULL'
```

**Comment by iamareebjamal** [comment]
Yes

**Comment by maze-runnar** [comment]
```
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.DuplicateAlias) table name "events" specified more than once

[SQL: UPDATE events SET is_cfs_enabled=False FROM (events INNER JOIN speakers_calls ON (events.id = speakers_calls.event_id AND speakers_calls.announcement IS NOT NULL))]
(Background on this error at: http://sqlalche.me/e/13/f405)

```

**Comment by iamareebjamal** [comment]
```sql
UPDATE events
SET    is_cfs_enabled = True,
FROM   events
       JOIN speakers_calls
         ON events.id = speakers_calls.event_id
        AND speakers_calls.announcement IS NOT NULL
```

This wasn't working?

**Comment by maze-runnar** [comment]
> ```sql
> UPDATE events
> SET    is_cfs_enabled = True,
> FROM   events
>        JOIN speakers_calls
>          ON events.id = speakers_calls.event_id
>         AND speakers_calls.announcement IS NOT NULL
> ```
> 
> This wasn't working?

no, it's not working in migration file. giving table name "events" specified more than once

**Comment by iamareebjamal** [comment]
```sql
UPDATE events
SET is_cfs_enabled = true
FROM speakers_calls
WHERE events.id = speakers_calls.event_id 
AND speakers_calls.announcement IS NOT NULL;
```

**Comment by maze-runnar** [comment]
ok

File: migrations/versions/rev-2021-01-15-13:41:59-a742abacc9b6_.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+revision = 'a742abacc9b6'
+down_revision = 'a3fb59fea6c2'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('groups',
+    sa.Column('deleted_at', sa.DateTime(timezone=True), nullable=True),
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
```
Comment (maze-runnar) [inline migrations/versions/rev-2021-01-15-13:41:59-a742abacc9b6_.py:24]: @iamareebjamal why this migration file is created on migration? 🙄

**Comment by iamareebjamal** [comment]
Because you are not rebased

**Comment by iamareebjamal** [comment]
@maze-runnar Update?

**Comment by maze-runnar** [comment]
I had deleted old fork. So do I have to create new PR?

**Comment by iamareebjamal** [comment]
Why you deleted the old fork?

**Comment by iamareebjamal** [comment]
We fixed the bug yesterday itself!

**Comment by maze-runnar** [comment]
Actually I had deleted that fork just before yesterday, just before that meeting.

**Comment by iamareebjamal** [comment]
Recreate the PR

**Pull request closed without merging**